lib/remote/filterutility.cpp

```diff
@@ -124,59 +124,92 @@ static void FilteredAddTarget(ScriptFrame& permissionFrame, Expression *permissi
 	}
 }
 
-void FilterUtility::CheckPermission(const ApiUser::Ptr& user, const String& permission, Expression **permissionFilter)
+static bool MatchPermission(const ApiUser::Ptr& user, const String& permission, Expression **permissionFilter = nullptr)
 {
 	if (permissionFilter)
 		*permissionFilter = nullptr;
 
 	if (permission.IsEmpty())
-		return;
+		return true;
 
 	bool foundPermission = false;
-	String requiredPermission = permission.ToLower();
```

Comment:
Was there any problem with this variable?

Reply:
This was purely to cache the passed permission parameter, which is actually unnecessary, instead of naming the new variable differently in the for loop.

Reply:
Why should one cache this? `requiredPermission` is nowhere used except at the end of the function when logging a warning.

Reply:
Nowhere else? Are you sure?

Reply:
reverted!

Reply:
By the way please also revert the other just-style changes. I'd like to see just what matters in _Files changed_ at least after hiding whitespace-only changes.